www_src/pages/section-2/section-2.jsx

```diff
@@ -295,6 +314,13 @@ var App = React.createClass({
   changeZoom: function (event) {
     this.refs.masterGrid.zoom(event.amount);
   },
+  showOverview: function () {
+    this.refs.masterGrid.setState({
```

Comment:
How is that better? I mean...it's more abstracted, but `setState` is a setter, no?